npm/deploy/Deployer.kt

```diff
@@ -26,6 +26,7 @@ import com.vaticle.bazel.distribution.common.Logging.LogLevel.DEBUG
 import com.vaticle.bazel.distribution.common.OS.MAC
```

Comment:
Update `deploy_npm` implementation, marking the auth token as a non-printable parameter, so it will be masked in the rule output.